src/Sylius/Bundle/ThemeBundle/.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+
+before_script: composer install --prefer-source --no-interaction
```

Comment:
Sure thing. In fact, we need to refactor our `.travis.yml` files as installing Composer dependencies or disabling XDebug should be included in `install` / `before_install` steps.